package.nls.json

```diff
@@ -589,5 +589,6 @@
     "AWS.walkthrough.gettingStarted.description": "These walkthroughs help you set up the AWS Toolkit for Visual Studio Code.",
     "AWS.walkthrough.gettingStarted.connect": "Connect to AWS",
     "AWS.walkthrough.gettingStarted.changeRegions": "Change AWS Regions",
-    "AWS.walkthrough.gettingStarted.setupToolchain": "Configure your toolchain"
+    "AWS.walkthrough.gettingStarted.setupToolchain": "Configure your toolchain",
+    "AWS.setting.togglePrompts.description": "Toggle prompts and notifications."
```

Comment:
Toggle implies that you can turn these messages off. I'd suggest something like "Restore all \"Do not show again\" notifications"

Reply:
Changed to "List of prompts not to show again"  This can be reset in the settings page if the user hovers over the setting where a little gear pops up which is a menu.  They can 'Reset setting' which  deletes the array from the json file which works fine. The setting will be added anytime the `disable` method is called.